akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -684,6 +689,7 @@ abstract class ShardCoordinator(
   var waitingForLocalRegionToTerminate = false
   var aliveRegions = Set.empty[ActorRef]
   var regionTerminationInProgress = Set.empty[ActorRef]
+  var waitingForRebalanceAcks = Map.empty[ShardId, Set[ActorRef]]
```

Comment:
Maybe this should be named `waitingForShardStopped` since it's only used by the StopShards?